test/collections/SortedVectorListTest.ooc

```diff
@@ -0,0 +1,74 @@
+/* This file is part of magic-sdk, an sdk for the open source programming language magic.
+ *
+ * Copyright (C) 2016 magic-lang
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+use base
+use collections
+use unit
+
+SortedVectorListClassTester: class {
+	_value: Float
+	value ::= this _value
+	init: func (=_value)
+}
+
+sortedVectorListIntComparator := func (a, b: Int*) -> Bool { a@ > b@ }
+sortedVectorListClassComparator := func (a, b: SortedVectorListClassTester*) -> Bool { a@ value > b@ value }
```

Comment:
@niklasborwell The problem is here. Change both `>` to `<`, and the test goes through.
The reason is that, in `primaryFeature1.4`, `VectorList sort()` has changed its argument from expecting a `greaterThan` comparator to an `isLess` comparator, in order to be consistent with other sorting-related methods.